web/src/signalsHandlers.js

```diff
@@ -105,74 +108,97 @@ const pickCrudAction = (entryTypeName, actionType) => {
   return actionSet[actionName]
 }
 
-export default (store) => (signal) => {
-  const { cellId } = signal.data
-  let { payload } = signal.data
-  // TODO: update holochain-conductor-api to latest
-  // which should deserialize this automatically
-  payload = msgpack.decode(payload)
+export default (store) => 
+{
 
-  // switch to CamelCasing if defined
-  const crudType = crudTypes[payload.entryType]
-  if (crudType) {
-    const action = pickCrudAction(crudType, payload.action)
-    store.dispatch(createSignalAction(action, cellId, payload.data))
-    // in case of the special situation with archiving edges
-    // where the layout reflow doesn't happen automatically
-    // we have to manually trigger it to do so
-    // the other cases are covered in src/layout/middleware.js ->
-    // isOneOfLayoutAffectingActions()
-    if (action.success().type === edgeActions.archiveEdge.success().type) {
-      store.dispatch(triggerUpdateLayout())
+  const timerIds = {}
+  return (signal) => {
+    const { cellId } = signal.data
+    let { payload } = signal.data
+    // TODO: update holochain-conductor-api to latest
+    // which should deserialize this automatically
+    payload = msgpack.decode(payload)
+  
+    if (payload.signalType === nonEntrySignalTypes.RealtimeInfo) {
+      console.log('received realtime signal:', payload.data)
+      // set timeout, and end any existing timeouts
```

Comment:
let's adjust this descriptive comment to not repeat the code "set timeout", but describe how this implements the feature we intend for